### PR TITLE
6485 display owner

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**

https://github.com/gravitee-io/issues/issues/6485

**Description**

A small description of what you did in that PR.

**Additional context**

For promote : 
image

For card-full
image

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A beta version of this package has been published! 🚀

**This is still experimental ⚠️**

To use it in your project run the following lines:
```shell
echo @gravitee-io:registry=https://npm.pkg.github.com > .npmrc
npm install @gravitee-io/ui-components@3.22.0-4c99a4d
rm -rf node_modules/@gravitee
mv node_modules/@gravitee-io node_modules/@gravitee
```

**Review by gcusnieux — APPROVED** [review]

**Comment by gaetanmaisse** [comment]
It looks like the icon isn't really align with the text:
image

File: src/molecules/gv-card-full.js [hunk 1]
```diff
@@ -176,12 +185,16 @@ export class GvCardFull extends ItemResource(LitElement) {
...
     const title = getTitle(this._item);
+    const owner = getOwner(this._item);
     const classes = { error: this._error || this._empty, card: true };
     return html`<div class="${classMap(classes)}" title="${title}" @click="${this._onClick}">
       <div class="${classMap({ skeleton: this._skeleton })}">
         <div class="${classMap({ image: true })}">${this._renderImage()}</div>
         <div class="content">
           <div class="${classMap({ title: true })}">${title}</div>
+          ${owner != null && owner.trim().length > 0
+            ? html`<div class="owner"><gv-icon shape="general:user" size="8px"></gv-icon>${owner}</div>`
```
Comment (gaetanmaisse) [inline src/molecules/gv-card-full.js:196]: Maybe some extra style is needed to align the icon

**Review by gaetanmaisse — CHANGES_REQUESTED** [review]

File: src/molecules/gv-card-full.js (hunk 1 above)
Comment (gcusnieux) [reply src/molecules/gv-card-full.js:196]: 👍 👓

Comment (ThibaudAV) [reply src/molecules/gv-card-full.js:196]: Thanks
better ? 
image

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A beta version of this package has been published! 🚀

**This is still experimental ⚠️**

To use it in your project run the following lines:
```shell
echo @gravitee-io:registry=https://npm.pkg.github.com > .npmrc
npm install @gravitee-io/ui-components@3.22.0-431c9a7
rm -rf node_modules/@gravitee
mv node_modules/@gravitee-io node_modules/@gravitee
```

**Review by gaetanmaisse — APPROVED** [review]

**Pull request merged**